components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -0,0 +1,206 @@
+import org.wso2.siddhi.query.api.definition.AbstractDefinition;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.query.api.execution.ExecutionElement;
+import org.wso2.siddhi.query.api.execution.query.Query;
+import org.wso2.siddhi.query.compiler.SiddhiCompiler;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SiddhiTryItClient {
+
+	private static Log log = LogFactory.getLog(SiddhiTryItClient.class);
+	private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+	private String errMsg = "";
+
+	/**
+	 * Event stream will be processed according to the specified execution plan
+	 *
+	 * @param executionPlan execution plan for siddhi query processing
+	 * @param eventStream   event stream
+	 * @param dateTime      date and time to begin the process
+	 */
+	public Map<String, StringBuilder> processData(String executionPlan, String eventStream,
+	                                              String dateTime) throws Exception {
+
+		Map<String, StringBuilder> map = new LinkedHashMap<String, StringBuilder>();
+		long beginSetTime = createTimeStamp(dateTime);
+		long beginSystemTime = System.currentTimeMillis();
+
+		// Create Siddhi Manager
+		SiddhiManager siddhiManager = new SiddhiManager();
+		ExecutionPlan newExecutionPlan = SiddhiCompiler.parse(executionPlan);
+		ExecutionPlanRuntime executionPlanRuntime =
+				siddhiManager.createExecutionPlanRuntime(executionPlan);
+
+		//Query Callback
+		for (int i = 0; i < newExecutionPlan.getExecutionElementList().size(); i++) {
+			String queryName = "";
+			ExecutionElement executionElement = newExecutionPlan.getExecutionElementList().get(i);
+
+			if(executionElement instanceof Query)   {
+				Query query = (Query) executionElement;
+				if (query.getAnnotations().size() > 0) {
+					queryName = query.getAnnotations().get(0).getElement("name");
+					if (!queryName.equals("")) {
+						final StringBuilder stringBuilder = new StringBuilder();
+						map.put(queryName, stringBuilder);
+						executionPlanRuntime.addCallback(queryName, new QueryCallback() {
+							@Override public void receive(long timeStamp, Event[] inEvents,
+							                              Event[] removeEvents) {
+								stringBuilder.append(gson.toJson(inEvents));
+							}
+						});
+					}
+				}
+			}
+		}
+
+		//Stream Callback
+		for(AbstractDefinition abstractDefinition:executionPlanRuntime.getStreamDefinitionMap().values()){
+			String streamName=abstractDefinition.getId();
+			final StringBuilder stringBuilder = new StringBuilder();
+			map.put(streamName, stringBuilder);
+			executionPlanRuntime.addCallback(streamName, new StreamCallback() {
+				@Override public void receive(Event[] events) {
+					stringBuilder.append(gson.toJson(events));
+				}
+			});
+		}
+
+		Pattern eventPattern = Pattern.compile("(\\S+)=\\[(.*)\\]");
```

Comment:
@prabathabey 
Thank you for the suggestion. Fixed in PR: https://github.com/wso2/carbon-event-processing/pull/181